src/pages/HomePage/HomePage.style.ts

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+
+const HomePageContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`;
```

Comment:
`justify-content: center` 속성에 의해서

페이지 너비가 줄어들어도 가운데로 정렬되고 있는 현상이 있어서
스크롤이 가능하게 설정해주는게 필요할 것 같습니다!
양 옆에 여백도 주면 좋을 것 같아요~

```suggestion
const HomePageContainer = styled.div`
  display: flex;
  padding: 0 2rem;
  overflow: auto;
`;
```

Reply:
고정된 페이지 너비값만 생각하고 작성했던 부분을 잘 잡아주셨네요 👍